File: Unidade I/Lista/Lista Estática Circular/Lista Estática Java/Listavel.java
public interface Listavel {
    // Métodos Fundamentais
    void anexar(Object dado);                   // c
    void inserir(int posicao, Object dado);     // c
    Object selecionar(int posicao);             // r
    Object[] selecionarTodos();                 // r
    void atualizar(int posicao, Object dado);   // u
    Object apagar(int posicao);                 // d

    // Métodos auxiliares
    boolean estaCheia();
    boolean estaVazia();
    String imprimir();
}

File: Lista/Lista Estática Circular/Lista Estática Kotlin/Listavel.kt
interface Listavel {
    // Metodos Fundamentais
    fun anexar(dado: Any?)                  //c
    fun inserir(posicao: Int, dado: Any?)   //c
    fun selecionar(posicao: Int): Any?      //r
    fun selecionarTodos(): Array<Any?>      //r
    fun atualizar(posicao: Int, dado: Any?) //u
    fun apagar(posicao: Int): Any?          //d

    // Metodos auxiliares
    fun estaCheia(): Boolean
    fun estaVazia(): Boolean
    fun imprimir(): String
}